Makefile

```diff
@@ -55,11 +45,12 @@ build: binary-build
 	@$(call check-static-binary,deis)
 
 build-all:
-	${DEV_ENV_CMD} gox -verbose \
-	${GO_LDFLAGS} \
-	-os="linux darwin " \
-	-arch="amd64 386" \
-	-output="$(DIST_DIR)/deis-${VERSION}-{{.OS}}-{{.Arch}}" .
+	${DEV_ENV_CMD} gox -verbose ${GO_LDFLAGS} -os="linux darwin " -arch="amd64 386" -output="$(DIST_DIR)/deis-latest-{{.OS}}-{{.Arch}}" .
```

Comment:
Answers inline:


1. _update/override the existing deis-latest-{{.OS}}-{{.Arch}} binary with this latest version_ - correct
2. _throw latest version into the ${TRAVIS_TAG} subdir <--- What if this env var is not set/does not exist? (It will only exist if the travis build is kicked off from pushing a git tag to the repo, right?)_ - this env var is not set, we'll fall back to just using `${VERSION}`, which is still `$(shell git rev-parse --short HEAD)`
3. _throw latest version into the ${VERSION} subdir_ - as mentioned in the previous point, the Makefile should only be doing this if `TRAVIS_TAG` isn't set. Did I get my `ifndef` blocks wrong perhaps?